pkgs/by-name/sw/swayfx-unwrapped/package.nix

```
@@ -57,6 +58,12 @@ stdenv.mkDerivation (finalAttrs: {
     [
       ./load-configuration-from-etc.patch
 
+      (fetchpatch2 {
+        # fix missing switch statement for newer libinput
+        url = "https://github.com/swaywm/sway/pull/8470.patch";
```

```suggestion
        url = "https://github.com/swaywm/sway/pull/8470.patch?full_index=1";
```

(and hash will have to be updated, naturally)

---

done